crawler.py

```diff
@@ -13,7 +13,7 @@
 
 # Configuration via environment variables
 NUM_BROWSERS = int(os.getenv('NUM_BROWSERS', '1'))
```

Comment:
Due to https://github.com/mozilla/OpenWPM/pull/468#discussion_r315900063, we should probably remove this env var and set it to 1.

Reply:
Done